packages/markers/src/browser/problem/problem-tree-model.spec.ts

```diff
@@ -125,6 +125,15 @@ describe('Problem Tree', () => {
             expect(problemTree['sortMarkers'](nodeB, nodeA)).equals(10);
         });
 
+        it('should sort markers based on owner if their severities, line numbers and columns are equal', () => {
+            const markerA = createMockMarker({ start: { line: 1, character: 10 }, end: { line: 1, character: 10 }, }, DiagnosticSeverity.Error, 'A');
+            const markerB = createMockMarker({ start: { line: 1, character: 10 }, end: { line: 1, character: 10 } }, DiagnosticSeverity.Error, 'B');
```

Comment:
minor: remove extra comma so everything is aligned :)

```suggestion
            const markerA = createMockMarker({ start: { line: 1, character: 10 }, end: { line: 1, character: 10 } }, DiagnosticSeverity.Error, 'A');
            const markerB = createMockMarker({ start: { line: 1, character: 10 }, end: { line: 1, character: 10 } }, DiagnosticSeverity.Error, 'B');
```